Cargo.toml

```diff
@@ -111,11 +111,11 @@ witness-generator = { path = "crates/witness-generator" }
 zkevm-metrics = { path = "crates/metrics" }
 benchmark-runner = { path = "crates/benchmark-runner" }
 
-zkvm-interface = { git = "https://github.com/eth-applied-research-group/ere", package = "zkvm-interface" }
-ere-sp1 = { git = "https://github.com/eth-applied-research-group/ere", package = "ere-sp1" }
-ere-risczero = { git = "https://github.com/eth-applied-research-group/ere", package = "ere-risczero" }
-ere-pico = { git = "https://github.com/eth-applied-research-group/ere", package = "ere-pico" }
-ere-openvm = { git = "https://github.com/eth-applied-research-group/ere", package = "ere-openvm" }
+zkvm-interface = { git = "https://github.com/eth-applied-research-group/ere", rev = "5f4fe5e7a83c969cab160b10644330ab771ee94c", package = "zkvm-interface" }
```

Comment:
I guess at some point we can start creating tags.

Reply:
Yep, and or publishing to crates.io